### PR TITLE
Display consistent log output between cuda and opencl

### PR DESCRIPTION
**Pull request by jean-m-cyr**
Also make switch time output dependent on log verbosity
level of 6 or greater.

### CODE REVIEW

**Comment by jean-m-cyr** [comment]
It would take a 32 GPU rig, each GPU running at 135MH/S, for the whole rig to overflow a 32-bit counter!

File: libethcore/Miner.h [hunk 1]
```diff
@@ -229,7 +229,7 @@ class Miner: public Worker
 
 	WorkPackage work() const { Guard l(x_work); return m_work; }
 
-	void addHashCount(uint64_t _n) { m_hashCount.fetch_add(_n, std::memory_order_relaxed); }
+	void addHashCount(uint32_t _n) { m_hashCount.fetch_add(_n, std::memory_order_relaxed); }
```
Comment (chfast) [inline libethcore/Miner.h]: Why the change?

Comment (jean-m-cyr) [reply libethcore/Miner.h]: Called by individual card once per reporting interval. 32 bits will hold up to 4gigqa hashes. Should be adequate for whole rig, and is passed in single 32 bit register.

Comment (jean-m-cyr) [reply libethcore/Miner.h]: 32 bits is enough for 2^32 hashes during reporting interval. Even the fastest card would take nearly 2 minutes to overflow. I'll change it back to 64 bits, as well as the global counter. It is negligible performance penalty.

Comment (jean-m-cyr) [reply libethcore/Miner.h]: The fastest available card would take a few minutes to overflow, and clminer only passes a 32 bit value is passed in, so there never was a need for 32 bits here.

Comment (chfast) [reply libethcore/Miner.h]: But what was the problem with uint64_t. Switching to uint32_t can make this actually slower.

Comment (jean-m-cyr) [reply libethcore/Miner.h]: No problem really, but both clminer and cudaminer use 32-bit counters locally, which is then passed as parameter to this function call. I've restored the per miner counter that adds up these successive calls to 64-bits, so no risk of overflow, and the conversion to 64-bit only happens in one place, inside the addHashCount function.

Comment (jean-m-cyr) [reply libethcore/Miner.h]: The workLoop in both miners use a 32-bit counter to accumulate hash count

Comment (jean-m-cyr) [reply libethcore/Miner.h]: Nevermind, I just noticed that cudaminer has 64-bit counter locally. I'll change back to 64 bit, and also update clminer to 64 bits

**Comment by jean-m-cyr** [comment]
Ok, all back to 64-bits. Rebasing to single commit.

**Comment by jean-m-cyr** [comment]
Closing, will resubmit as single commit

**Comment by chfast** [comment]
You don't need a new PR for that. You can force push to the same branch.

**Comment by jean-m-cyr** [comment]
I tried that and it kept retaining all of the previous commits here... but not locally!
